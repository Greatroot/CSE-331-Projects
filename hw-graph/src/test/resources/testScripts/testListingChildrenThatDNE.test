# Tests listing children for a one node graph with no edges
 | should return no children.

CreateGraph graph
AddNode graph n1
ListChildren graph n1