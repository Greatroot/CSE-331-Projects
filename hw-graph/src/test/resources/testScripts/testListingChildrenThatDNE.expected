# Tests listing children for a one node graph with no edges | should return no children.

created graph graph
added node n1 to graph
the children of n1 in graph are: